fibers.texi

```diff
@@ -1389,11 +1389,11 @@ Guile now has a solid concurrency story.  Use fibers, incorporate it
 directly into your project, fork it, improve it: what happens now is
 up to you.  Happy hacking and godspeed!
 
-@c @node Concept Index
-@c @unnumbered Concept Index
-@c @printindex cp
-@c @node Function Index
-@c @unnumbered Function Index
-@c @printindex fn
+@node Concept Index
+@unnumbered Concept Index
```

Comment:
I think including an index while it's still empty is a bit silly, but IMO go ahead with the rest!
I would like some confirmation from another reviewer on the location though.

Reply:
(it can easily be re-added once there's actually some things inside)